CONTRIBUTING.rst

```diff
@@ -13,7 +13,7 @@ Types of Contributions
 Report Bugs
 ~~~~~~~~~~~
 
-Report bugs at https://github.com/pcdshub/ads-async/issues.
+Report bugs at https://github.com/pcdshub/ads_async/issues.
```

Comment:
Are we moving the repo name?

Reply:
_Nooo_ - I messed this up on my side. Thanks for catching it.